index.html

```diff
@@ -1,19 +1,125 @@
+    <link href="https://fonts.googleapis.com/css?family=Charmonman" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+
+    <div class="container">
+
+      <nav>
+
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+
+        <a href="index.html" class="logo"><img src="img/logo.png" alt="Great Idea! Company logo."></a>
+
+      </nav>
+
+      <header class="homeBanner">
+
+        <div class="header-text">
+
+          <h1><br>Innovation<br>On<br>Demand</h1><br>
+
+          <button class="startButton">Get Started</button>
+
+        </div>
+
+        <img class="header-img" src="img/header-img.png" alt="Image of a code snippet.">
+
+      </header>
+
+      <div class="section1">
+
+        <section>
+
+          <h2>Features</h2>
+
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+        </section>
+
+        <section>
+
+          <h2>About</h2>
+
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+        </section>
+
+      </div>
+
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+      <div class="section2">
+
+        <section>
+
+          <h2>Services</h2>
+
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+        </section>
+
+        <section>
+
+          <h2>Product</h2>
+
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+        </section>
+
+        <section>
+
+          <h2>Vision</h2>
+
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+        </section>
+
+      </div>
+
+      <section class="contact">
+
+        <h2>Contact</h2>
+
+        <address>
+
+          123 Way 456 Street<br>
+          Somewhere, USA<br><br>
+          1 (888) 888-8888<br><br>
+          sales@greatidea.io<br>
+
+        </address>
+
+      </section>
+
+      <footer class="pg1footer">
+
+        <p><i>Copyright Great Idea! 2018</i></p>
```

Comment:
I think this line (_117_) is fair, but consider separating out elements onto separate lines for readability:
```html
<p>
  <i>Copyright Great Idea! 2018</i>
<p>
```